flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/BoltWrapperTest.java

```diff
@@ -265,12 +264,12 @@ public void testOpen() throws Exception {
 	@Test
 	public void testOpenSink() throws Exception {
 		final IRichBolt bolt = mock(IRichBolt.class);
-		BoltWrapper<Object, Object> wrapper = new BoltWrapper<Object, Object>(bolt);
+		BoltWrapper<Object, Object> wrapper = new BoltWrapper<Object, Object>(bolt, "stream", "component");
 		
 		wrapper.setup(createMockStreamTask(), new StreamConfig(new Configuration()), mock(Output.class));
 		wrapper.open();
 		
-		verify(bolt).prepare(any(Map.class), any(TopologyContext.class), any(OutputCollector.class));
+		verify(bolt).prepare(any(Map.class), any(TopologyContext.class), isNotNull(OutputCollector.class));
```

Comment:
`any` matches also null (literally anything) but here I want to explicitly check `isNotNull`.